### PR TITLE
Update BOTMETA.yml migrated_to for Netbox components

### PR DESCRIPTION
**Pull request by FragmentedPacket**
##### SUMMARY

Updated BOTMETA.yml to show that the Netbox modules have been moved to Ansible Collections.

Is there anything else I need to do before all the migrations start to occur?

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

netbox.py
netbox_device.py
netbox_interface.by
netbox_site.py
netbox_ip_address.py
netbox_prefix.py

### CODE REVIEW

File: .github/BOTMETA.yml
Comment (gundalow) [inline .github/BOTMETA.yml]: `migrated_to` needs to be against every file, not the directory. See other examples in this file.

Comment (FragmentedPacket) [reply .github/BOTMETA.yml]: This has been changed.

**Comment by FragmentedPacket** [comment]
@gundalow I've fixed the tests so they pass.

Does that look correct now?

**Review by gundalow — APPROVED** [review]

**Comment by abenokraitis** [comment]
`netbox_community.ansible_modules` IMO is not the most user friendly selection. Why not just use `netbox.netbox` if you know you'll only have one collection with all the modules/content?

**Comment by FragmentedPacket** [comment]
I don't want users to get confused with an official "NetBox" collection and signify this is a NetBox community driven collection. They also have the Github repo `netbox-community` so I'm trying to stick with the way they're trying to keep things signified as community driven.

**Comment by abenokraitis** [comment]
@FragmentedPacket Can you elaborate the difference between an "official" collection and a community one? For example, is a corporate entity running the official collection, or is the codebase completely different?

**Comment by abenokraitis** [comment]
If NTC is going to handle their own "official" branch I'd recommend having `netbox.netbox` be the upstream community collection and `networktocode.netbox` be the downstream collection if that's the goal/intent.

**Comment by FragmentedPacket** [comment]
@abenokraitis Sorry that wasn't very clear. I do not own the NetBox name or any rights to it and rather not infringe upon the maintainer's ability to create their own collection if they deem necessary or have it appear that this collection is supported by NetBox.

Currently there is no `netbox` collection, but I don't feel like it's my right to own that namespace.

NetBox is an open source software so no corporate entity owns it.

This collection isn't associated to either NetBox or NTC. It's my own project. I can ask the maintainer's if I can own that `netbox` namespace on Ansible Galxy, but this is purely a community driven collection.

**Comment by abenokraitis** [comment]
If the netbox community allows you, I would say go ahead and do it and then you may have a community of maintainers of the Galaxy namespace, same as the modules and collections. There are other examples of this in the larger community, so this wouldn't be a huge stretch. If the modules in the collection you are packaging are indeed the best example of netbox implementation I would lean towards a `netbox.netbox` and provide any support or quality claims in the readme.

**Comment by gundalow** [comment]
Just to clarify, the name of the GitHub repo doesn't need to be the same as the collection.
Also the GitHub Org doesn't need to match the Namespace.

**Comment by FragmentedPacket** [comment]
@gundalow Thank you. That was the reasoning the namespace was set as it is today. I thought when requesting it, it had to match a Github org.

I appreciate the clarity.

@abenokraitis Let me discuss it with the maintainers and see if they mind or not. If they don't mind, I will request the `netbox` namespace and push a release up to it, and then update this file.

**Comment by abenokraitis** [comment]
sounds good, it would make sense to go ahead and request the Ansible Galaxy Namespace as well for `netbox` as well.

https://github.com/ansible/galaxy/issues/new?template=New_namespace.md

**Comment by FragmentedPacket** [comment]
https://github.com/ansible/galaxy/issues/2277

I submitted a PR to add a new namespace. Once that is approved/created, I will make all the necessary code changes, and then push up a new release renaming to collection to `netbox.netbox`.

Thanks for the discussion.

**Comment by FragmentedPacket** [comment]
@gundalow @abenokraitis This has been updated for new FQCN and I also pushed a release to it so it's available in Galaxy now.

**Pull request merged**
